presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -523,7 +523,9 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateTaskImpl(
           planFragment,
           prestoTask->id.id(),
           std::move(queryCtx),
-          exec::Task::ExecutionMode::kParallel);
+          exec::Task::ExecutionMode::kParallel,
+          static_cast<exec::Consumer>(nullptr),
```

Comment:
hmm, we shall actually place this ahead of consumer and after execution mode.

Reply:
Sure. Let's make it a separate refactoring after this is done.